integration/mock-backend-server/src/main/java/org/wso2/choreo/connect/mockbackend/MockServices.java

```diff
@@ -32,10 +32,12 @@ public static void main(String[] args) {
         MockBackend2 mockBackend2 = new MockBackend2(Constants.MOCK_BACKEND2_SERVER_PORT);
         // TODO: (VirajSalaka) start analytics server only when it requires
         MockAnalyticsServer mockAnalyticsServer = new MockAnalyticsServer(Constants.MOCK_ANALYTICS_SERVER_PORT);
+        MockBackendSOAP mockBackendSOAP = new MockBackendSOAP();
```

Comment:
Shall we start the soap backend only if it is needed? Similar to https://github.com/wso2/product-microgateway/pull/2972/files#diff-97e9511198aca64a731c90c2c0e3b325a68e51ad0306fdb816d8f548de748b62L55

Reply:
Correct link https://github.com/wso2/product-microgateway/blob/2f383839f97220bd7abbd350dd4c53090ad0618a/integration/mock-backend-server/src/main/java/org/wso2/choreo/connect/mockbackend/MockServices.java#L55

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5c91048f9fa451e4b2f2d91dac704d3f6b755cd3